cuqi/implicitprior/_regularizedGaussian.py

```diff
@@ -189,12 +189,18 @@ def _parse_preset_constraint_input(self, constraint, optional_regularization_par
             elif c_lower == "increasing":
                 if not isinstance(self.geometry, Continuous1D):
                     raise ValueError("Geometry not supported for " + c_lower)
-                self._constraint_prox = lambda z, _: spoptimize.isotonic_regression(z, increasing=True).x
+                if hasattr(spoptimize, 'isotonic_regression'):
+                    self._constraint_prox = lambda z, _: spoptimize.isotonic_regression(z, increasing=True).x
+                else:
+                    raise AttributeError(f"The function 'isotonic_regression' does not exist in scipy.optimize. Installed scipy version: {spoptimize.__version__}.")
```

Comment:
I suggest informing the user with the solution, to install newer scipy version, e.g. 
```suggestion
                    raise AttributeError(f"The function 'isotonic_regression' does not exist in scipy.optimize. Installed scipy version: {spoptimize.__version__}. You need to install a newer version of scipy")
```